tests/scripts/thread-cert/border_router/MATN_03_InvalidCommissionerDeregistration.py

```diff
@@ -0,0 +1,185 @@
+    USE_MESSAGE_FACTORY = False
+
+    TOPOLOGY = {
+        BR_1: {
+            'name': 'BR_1',
+            'is_otbr': True,
+            'allowlist': [BR_2, ROUTER],
+            'version': '1.2',
+            'channel': CHANNEL1,
+            'router_selection_jitter': 2,
+        },
+        BR_2: {
+            'name': 'BR_2',
+            'allowlist': [BR_1, ROUTER],
+            'is_otbr': True,
+            'version': '1.2',
+            'channel': CHANNEL1,
+            'router_selection_jitter': 2,
+        },
+        ROUTER: {
+            'name': 'Router',
+            'allowlist': [BR_1, BR_2],
+            'version': '1.2',
+            'channel': CHANNEL1,
+            'router_selection_jitter': 2,
+        },
+        HOST: {
+            'name': 'Host',
+            'is_host': True
+        },
+    }
+
+    def test(self):
+        br1 = self.nodes[BR_1]
+        br2 = self.nodes[BR_2]
+        router = self.nodes[ROUTER]
+        host = self.nodes[HOST]
+
+        br1.start()
+        self.simulator.go(5)
+        self.assertEqual('leader', br1.get_state())
+        self.assertTrue(br1.is_primary_backbone_router)
+
+        router.start()
+        self.simulator.go(5)
+        self.assertEqual('router', router.get_state())
+
+        br2.start()
+        self.simulator.go(5)
+        self.assertEqual('router', router.get_state())
+
+        host.start(start_radvd=False)
+        self.simulator.go(5)
+
+        router.add_ipmaddr(MA1)
+        self.simulator.go(5)
+
+        # Ensure that router can receive a multicast ping packet sent to MA1.
+        self.assertTrue(
+            host.ping(MA1, backbone=True, ttl=10, interface=host.get_ip6_address(config.ADDRESS_TYPE.ONLINK_ULA)[0]))
+        self.simulator.go(5)
+
+        # 1. Router attempts to deregister for multicast address, MA1, at BR_1.
+        router.register_multicast_listener(MA1, timeout=0)
+
+        # 3. Host sends a ping packet to the multicast address, MA1.
+        self.assertTrue(
+            host.ping(MA1, backbone=True, ttl=10, interface=host.get_ip6_address(config.ADDRESS_TYPE.ONLINK_ULA)[0]))
+        self.simulator.go(5)
+
+        self.collect_ipaddrs()
+        self.collect_rloc16s()
+        self.collect_rlocs()
+        self.collect_leader_aloc(BR_1)
+        self.collect_extra_vars()
+
+    def verify(self, pv: pktverify.packet_verifier.PacketVerifier):
+        pkts = pv.pkts
+        vars = pv.vars
+        pv.summary.show()
+
+        # Ensure the topology is formed correctly
+        pv.verify_attached('Router', 'BR_1')
+        pv.verify_attached('BR_2')
+
+        # 2. BR_1 responds to the multicast registration successfully,
+        # ignoring the timeout value.
+        pkts.filter_wpan_src64(vars['BR_1']) \
+            .filter_ipv6_dst(vars['Router_RLOC']) \
+            .filter_coap_ack('/n/mr') \
+            .filter(lambda p: p.coap.code == 68 and p.thread_nm.tlv.status == 0) \
```

Comment:
```suggestion
            .filter(p.thread_nm.tlv.status == 0) \
```
`code == 68` already verified by `filter_coap_ack`